src/TableSchema.sql

```diff
@@ -63,4 +63,35 @@ INSERT INTO admin (username, password)
 VALUES ('admin', 'admin');
 
 INSERT INTO patient (first_name, last_name, username, password, phone_no, emaiL, birth_date, gender, address)
-VALUES ('Mugdha', 'Kurkure', 'mugs1234', 'mugs1234', '9008210569', 'mugdha.k@gmail.com', '2001-09-18', 'FEMALE', 'Thane');
\ No newline at end of file
+VALUES ('Mugdha', 'Kurkure', 'mugs1234', 'mugs1234', '9008210569', 'mugdha.k@gmail.com', '2001-09-18', 'FEMALE', 'Thane');
+
+INSERT INTO doctor (first_name, last_name, qualification)
+VALUES ('Mihir', 'Pandya', 'Obstretician');
+
+INSERT INTO doctor (first_name, last_name, qualification)
+VALUES ('Mugdha', 'Kurkure', 'Pediatrician');
+
+INSERT INTO doctor (first_name, last_name, qualification)
+VALUES ('Vani', 'Kamani', 'Nutritionist');
+
+-- query for getting reports and patient ID using date
+SELECT report.report_id, patient_report.patient_id, patient.first_name, patient.last_name, report.name, report.file
+FROM report
+JOIN patient_report ON report.report_id = patient_report.report_id
+JOIN patient ON patient.patient_id = patient_report.patient_id
+WHERE report.date = '2021-05-18';
+
+-- query for getting reports and date using patient ID
+SELECT report.report_id, report.date, patient.first_name, patient.last_name, report.name, report.file
+FROM report
+JOIN patient_report ON report.report_id = patient_report.report_id
```

Comment:
what's the diff between JOIN and INNER JOIN in sqlite3

Reply:
### The difference between INNER JOIN and a NATURAL JOIN:

- In INNER JOIN, you have to specify a join condition which the inner join uses to join the two tables. Whereas in the natural join, you don't write a join condition. You just write the two tables' names without any condition. Then the natural join will automatically test for equality between the values for every column exists in both tables. Natural join infers the join condition automatically.
- In the NATURAL JOIN, all the columns from both tables with the same name will be matched against each other. For example, if we have two tables with two column names in common (the two columns exists with the same name in the two tables), then the natural join will join the two tables by comparing the values of both columns and not just from one column.